### PR TITLE
Crm/tuanna/upload download

### PR DESCRIPTION
**Pull request by nineplusv01**
TuanNA add function upload-download

### CODE REVIEW

File: src/main/java/com/nineplus/bestwork/dto/FileStorageResDto.java [hunk 1]
```diff
@@ -35,9 +33,12 @@ public class FileStorageResDto extends BaseDto {
 	private String type;
 
 	@JsonProperty("createDate")
-	private Timestamp createDate;
+	private String createDate;
```
Comment (codefunli) [inline src/main/java/com/nineplus/bestwork/dto/FileStorageResDto.java:36]: Please roll back to timstamp

File: src/main/java/com/nineplus/bestwork/services/IPostInvoiceService.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.springframework.web.multipart.MultipartFile;
+
+import com.nineplus.bestwork.dto.AirWayBillAllInfoResDto;
+import com.nineplus.bestwork.dto.PostInvoiceReqDto;
+import com.nineplus.bestwork.dto.PostInvoiceResDto;
+import com.nineplus.bestwork.entity.PostInvoice;
+import com.nineplus.bestwork.exception.BestWorkBussinessException;
+
+public interface IPostInvoiceService {
```
Comment (codefunli) [inline src/main/java/com/nineplus/bestwork/services/IPostInvoiceService.java]: IPostInvoiceService => IInvoicePostService

File: src/main/java/com/nineplus/bestwork/services/impl/SftpFileServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+	 * The Constant SEPARATOR.
+	 */
+	public static final String SEPARATOR = "/";
+
+	public static final String HYPHEN = "-";
+
+	/**
+	 * The Constant ROOT_PATH.
+	 */
+	public static final String ROOT_PATH = "/home/bestwork";
```
Comment (codefunli) [inline src/main/java/com/nineplus/bestwork/services/impl/SftpFileServiceImpl.java:48]: Please use properties file to assign these values

File: src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java [hunk 4]
```diff
@@ -42,7 +42,7 @@ public FileStorageEntity storeFilePost(String imageData, PostEntity reqPost) {
 			image.setName(imageName);
 			String type = getImageType(imageData);
 			image.setType(type);
-			image.setCreateDate(Timestamp.valueOf(LocalDateTime.now(ZoneId.of("Asia/Ho_Chi_Minh"))));
```
Comment (codefunli) [inline src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java:45]: Please roll back to Timestamp type

File: src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java [hunk 5]
```diff
@@ -83,7 +84,7 @@ public FileStorageEntity storeFileProgress(FileStorageReqDto file, ProgressEntit
 			String generatedFileName = UUID.randomUUID().toString().replace("-", "");
 			image.setName(generatedFileName);
 			image.setType(getImageType(file.getData()));
-			image.setCreateDate(Timestamp.valueOf(LocalDateTime.now(ZoneId.of("Asia/Ho_Chi_Minh"))));
```
Comment (codefunli) [inline src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java:86]: Please roll back to Timestamp type

File: src/main/java/com/nineplus/bestwork/services/IPostInvoiceService.java (hunk 2 above)
Comment (nineplusv01) [reply src/main/java/com/nineplus/bestwork/services/IPostInvoiceService.java]: Changed name to  IInvoicePostService

File: src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java (hunk 4 above)
Comment (nineplusv01) [reply src/main/java/com/nineplus/bestwork/services/impl/StorageServiceImpl.java:45]: Rolled back to timestamp

File: src/main/java/com/nineplus/bestwork/services/impl/SftpFileServiceImpl.java (hunk 3 above)
Comment (nineplusv01) [reply src/main/java/com/nineplus/bestwork/services/impl/SftpFileServiceImpl.java:48]: Use property setting in application.properties

**Review by codefunli — APPROVED** [review]

**Pull request merged**